.gitignore

```diff
@@ -118,3 +118,32 @@ env/
 
 ### Configuration files ###
 .env
+
+# Local .terraform directories
+**/.terraform/*
```

Comment:
This looks like you've taken a standard .gitignore and also applied it to your .dockerignore which is a very sensible policy but it's worth being aware that despite the similarity there are some syntax differences between them which can catch you out - [see e.g. this article for examples](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/). I _believe_ they'll both work with the **/<directory/* syntax but I'm not certain of that, the way it catches me out is the inverse where .gitignore would also accept <directory> but .dockerignore won't. Nothing to change if it's working, but thought worth mentioning!